src/enums/web3.ts

```diff
@@ -0,0 +1,21 @@
+export enum ChainID {
+  Ethereum = 1,
+  Ropsten = 3,
+}
```

Comment:
Can we also add chain ID for the local Ganache?

```suggestion
export enum ChainID {
  Ethereum = 1,
  Ropsten = 3,
  Localhost = 1337,
}
```